test/common/packagelib.py

```diff
@@ -134,6 +136,13 @@ def setUp(self):
         # HACK: kpatch check sometimes complains that we don't set up a full repo in unrelated tests
         self.allow_browser_errors("Could not determine kpatch packages:.*repodata updates was not complete")
 
+    @cached_property
+    def getSystemdUnitsInfo(self):
+        return json.loads(self.machine.execute("systemctl list-units -la --plain -o json"))
+
+    def getSystemdExistingUnits(self):
+        return [foo["unit"] for foo in self.getSystemdUnitsInfo()]
```

Comment:
This feels a little misplaced in packagelib.py, but we can move it if/when we need/want.

Reply:
sure, I've also looked for some `service/systemdlib.py` or something :-) but finally also thouh that there is no currently  other better existing place.